elide-integration-tests/src/test/java/com/yahoo/elide/auditTests/AuditIT.java

```diff
@@ -6,110 +6,230 @@
+                        ObjectMapperType.GSON
+                )
                 .patch("/auditEntity/1")
                 .then()
                 .statusCode(HttpStatus.SC_NO_CONTENT);
 
-        Assert.assertTrue(logger.logMessages.contains("Updated relationship (for id: 1): 2"));
-        Assert.assertTrue(logger.logMessages.contains("Updated value (for id: 1): updated value"));
+        assertTrue(logger.logMessages.contains("Updated relationship (for id: 1): 2"));
+        assertTrue(logger.logMessages.contains("Updated value (for id: 1): updated value"));
     }
 
-    @Test(priority = 2)
+    @Test
     public void testAuditWithDuplicateLineageEntry() {
-        String request = jsonParser.getJson("/AuditIT/updateAuditEntityLineageDup.req.json");
+        // create auditEntity 1 and 2
+        createAuditEntity(AUDIT_1);
+        createAuditEntity(AUDIT_2);
 
+        // update auditEntity 1 through the relationship of auditEntity 2
         given()
                 .contentType(JSONAPI_CONTENT_TYPE)
                 .accept(JSONAPI_CONTENT_TYPE)
-                .body(request)
+                .body(
+                        datum(
+                                resource(
+                                        type("auditEntity"),
+                                        id("1"),
+                                        attributes(
+                                                attr("value", "update id 1 through id 2")
+                                        )
+                                )
+                        ),
+                        ObjectMapperType.GSON
+                )
                 .patch("/auditEntity/2/otherEntity/1")
                 .then()
                 .statusCode(HttpStatus.SC_NO_CONTENT);
 
-        Assert.assertTrue(logger.logMessages.contains("Updated value (for id: 1): update id 1 through id 2"));
+        assertTrue(logger.logMessages.contains("Updated value (for id: 1): update id 1 through id 2"));
     }
 
-    @Test(priority = 3)
+    @Test
     public void testAuditUpdateOnInverseCollection() {
-        String request = jsonParser.getJson("/AuditIT/createAuditEntityInverse.req.json");
+        // create auditEntity 1 and 2, update auditEntity 1 to have relationship to auditEntity 2
+        createAuditEntity(AUDIT_1);
+        createAuditEntity(AUDIT_2);
+        given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .body(
+                        datum(AUDIT_1_RELATIONSHIP),
+                        ObjectMapperType.GSON
+                )
+                .patch("/auditEntity/1")
+                .then()
+                .statusCode(HttpStatus.SC_NO_CONTENT);
 
-        Assert.assertFalse(logger.logMessages.contains("Inverse entities: [Value: update id 1 through id 2 relationship: 2]"));
+        assertFalse(logger.logMessages.contains("Inverse entities: [Value: updated value relationship: 2]"));
```

Comment:
The log message here is changed because this test doesn't depend on `testAuditWithDuplicateLineageEntry()` anymore.